aws-redshiftserverless-workgroup/src/main/java/software/amazon/redshiftserverless/workgroup/UpdateHandler.java

```diff
@@ -113,13 +123,24 @@ private ResourceModel getUpdatableResourceModel(ResourceModel desiredModel, Reso
                 .build();
     }
 
-    private GetWorkgroupResponse readWorkgroup(final GetWorkgroupRequest awsRequest,
-                                               final ProxyClient<RedshiftServerlessClient> proxyClient) {
-        GetWorkgroupResponse awsResponse;
-        awsResponse = proxyClient.injectCredentialsAndInvokeV2(awsRequest, proxyClient.client()::getWorkgroup);
+    private boolean isWorkgroupStable(final Object awsRequest,
```

Comment:
we are recreating this method multiple times, can we move them to the base class and use only one?

Reply:
Will do